receiver/postgresqlreceiver/templates/topQueryTemplate.tmpl

```diff
@@ -19,4 +19,5 @@ FROM
   LEFT JOIN pg_database ON pg_stat_statements.dbid = pg_database.oid
 WHERE
   query != '<insufficient privilege>'
+  AND query NOT LIKE '/* otel-collector-ignore */%'
```

Comment:
May I ask what is the `/* otel-collector-ignore */` for in this PR?

Reply:
in this pr a lot of queries are introduced, and most of them are not useful for the  `top query collection`. so we can just exclude them.